commands/http/handler.go

```diff
@@ -121,6 +121,16 @@ func (i internalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	i.ctx.Context = ctx
 	req.SetContext(i.ctx)
 
+	if cn, ok := w.(http.CloseNotifier); ok {
+		go func() {
+			select {
+			case <-cn.CloseNotify():
+				cancel()
+			case <-ctx.Done():
+			}
+		}()
+	}
+
```

Comment:
it looks like the http headers arent getting sent correctly... On the server side, i see the call being made to set the header for `X-Chunked-Content` but on the client side when we check the output, the header doesnt appear to be set.